### DISCUSSION TITLE
php artisan telebot:webhook No task specified!

### DISCUSSION THREAD

**Discussion by JekaSylar** [westacks/telebot#73] [Q&A] [answered]
Good afternoon
Sorry for the question, but there was a problem, I work locally through polling, everything is fine, messages are received and sent by the bot.
But when I uploaded the project to a real server and I want to run the webhook command
php artisan telebot:webhook
I get the error No task specified!
Package installed, cleared cache

**Comment by punyflash** [comment, marked as answer]
This library is open source and you can figure what are you doing wrong by yourself:
https://github.com/westacks/telebot/blob/c00ad32a35e0b5064ac343b08db601865852e283/src/Laravel/Artisan/TeleBotCommand.php#L23-L35

**Reply by JekaSylar** [reply]
Tell me what I need to do to start the webhook

**Reply by punyflash** [reply]
https://github.com/westacks/telebot#automatic-webhook-generation
